tailwind.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  mode: "jit",
+  purge: ["./pages/**/*.{js,ts,jsx,tsx}"],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    colors: {
+      transparent: "transparent",
+      current: "currentColor",
+      black: "#121212",
+      gray: {
+        DEFAULT: "#dedede",
+        dark: "#aaaaaa",
+      },
+    },
+    fontFamily: {
+      sans: ["Nunito", "sans-serif"],
```

Comment:
```suggestion
      sans: ["Nunito", ...defaultTheme.fontFamily.sans],
```

Based on [this documentation](https://tailwindcss.com/docs/theme#referencing-the-default-theme), I think you need to use `...defaultTheme.fontFamily.sans` instead of prescribing it into `"sans-serif"` so that we are still inheriting the sans serif font family defined by the default theme.

You can refer to [my commit here](https://github.com/zainfathoni/rumahberbagi-app/commit/11caa9dcbad3e75d802a7d8541925cafe18095f3#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2e) as a reference, thanks!